src/items/implementations.md

```diff
@@ -50,10 +50,44 @@ bracketed set of associable items.
 The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
-Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
-the associate item's path component. Inherent implementations cannot contain
-associated type aliases.
+Inherent implementations associate the contained items to the
+implementing type.  Inherent implementations can contain [associated
+functions] (including methods), [associated types], and [associated
```

Comment:
Associated types are type aliases, and as per the next statement, inherent impls cannot have them.

Reply:
Removed.